Palaso/Settings/CrossPlatformSettingsProvider.cs

```diff
@@ -14,17 +14,27 @@ namespace Palaso.Settings
 	/// </summary>
 	public class CrossPlatformSettingsProvider : SettingsProvider, IApplicationSettingsProvider
 	{
+		//Public only for unit testing. I can't get InternalsVisibleTo to work, possibly because of strong naming.
+		public const string UserConfigFileName = "user.config";
```

Comment:
Yes, including this one! Though I haven't found an instance of a test that depends on it. Both Resharper and full compilation report the methods as inaccessible.